crates/rpc/rpc-builder/tests/it/http.rs

```diff
@@ -44,43 +45,69 @@ where C: ClientT + SubscriptionClientT + Sync {
+    assert!(is_unimplemented(EthApiClient::block_by_number(client, BlockNumber::default(), false).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::block_transaction_count_by_hash(client, H256::default()).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::block_transaction_count_by_number(client, BlockNumber::default()).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::block_uncles_count_by_hash(client, H256::default()).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::block_uncles_count_by_number(client, BlockNumber::default()).await.err().unwrap()));
+    // EthApiClient::uncle_by_block_hash_and_index(client, H256::default(), Index::default()).await.err().unwrap()));
+    // EthApiClient::uncle_by_block_number_and_index(client).await.err().unwrap()));
+    // EthApiClient::transaction_by_hash(client, H256::default()).await.err().unwrap()));
+    // EthApiClient::transaction_by_block_hash_and_index(client).await.err().unwrap()));
+    // EthApiClient::transaction_by_block_number_and_index(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::transaction_receipt(client, H256::default()).await.err().unwrap()));
+    // EthApiClient::balance(client).await.err().unwrap()));
+    // EthApiClient::storage_at(client).await.err().unwrap()));
+    // EthApiClient::transaction_count(client).await.err().unwrap()));
+    // EthApiClient::get_code(client).await.err().unwrap()));
+    // EthApiClient::call(client).await.err().unwrap()));
+    // EthApiClient::create_access_list(client).await.err().unwrap()));
+    // EthApiClient::estimate_gas(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::gas_price(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::fee_history(client, U256::default(), BlockNumber::default(), None).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::max_priority_fee_per_gas(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::is_mining(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::hashrate(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::get_work(client).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::submit_hashrate(client, U256::default(), H256::default()).await.err().unwrap()));
+    assert!(is_unimplemented(EthApiClient::submit_work(client, H64::default(), H256::default(), H256::default()).await.err().unwrap()));
+    // EthApiClient::send_transaction(client).await.err().unwrap()));
+    // EthApiClient::send_raw_transaction(client).await.err().unwrap()));
+    // EthApiClient::sign(client).await.err().unwrap()));
+    // EthApiClient::sign_transaction(client).await.err().unwrap()));
+    // EthApiClient::sign_typed_data(client).await.err().unwrap()));
+    // EthApiClient::get_proof(client).await.err().unwrap()));
```

Comment:
why are they currently commented?

Reply:
@mattsse maybe you were looking at an older version of the PR?